.github/workflows/run_tests.yml

```diff
@@ -6,12 +6,14 @@ name: Run tests
 on: [push, pull_request]
```

Comment:
we probably don't need both. @Drulex @kousu which one is preferred?

Reply:
You want it on `pull_request` on all branches and `push` just on `master`. The syntax for this in Travis is different because Travis's `pull_request` option is a setting in their GUI

https://github.com/neuropoly/spinalcordtoolbox/blob/49cb4a16ade7c91533e669c8d1d2a777f5bbfe66/.travis.yml#L19-L22

I don't know off hand the syntax for that in Github Actions.

Reply:
I think something like this : 
~~~
on:
  # Trigger the workflow on push or pull request,
  # but only for the master branch
  push:
    branches:
      - master
  pull_request
~~~
Should work after reading https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions .
I am going to try this

Reply:
Ok si after getting it wrong a bunch of times I think the correct solution is in a863278